spec/lib/diaspora/shareable_spec.rb

```diff
@@ -17,5 +17,28 @@
         expect(Post.all_public.map(&:id)).to eq([])
       end
     end
+
+    context "having multiple objects with equal db IDs" do
+      before do
+        id = 60_000
```

Comment:
I also feel now that hardcoding this constant is also not too good style. I know how to rewrite it without hardcoding. Do you think I should do it?